imageroot/etc/state-include.conf

```diff
@@ -6,3 +6,5 @@
 
 
 state/netdata
+volumes/lib
+volumes/cache
```

Comment:
@DavidePrincipi I have merged your commit but I have a doubt, here we are root full and I create the volume 

```
    --volume ${MODULE_ID}-lib:/var/lib/netdata:Z \
    --volume ${MODULE_ID}-cache:/var/cache/netdata:Z \
```